gnovm/pkg/gnolang/preprocess.go

```diff
@@ -803,40 +816,44 @@ func Preprocess(store Store, ctx BlockNode, n Node) Node {
 								Op:    n.Op,
 								Right: rn,
 							}
-							resn := Node(Call(tx, n2))
+							resn := Node(Call(tx, n2)) // this make current node to gonative{xxx}
 							resn = Preprocess(store, last, resn)
 							return resn, TRANS_CONTINUE
 							// NOTE: binary operations are always computed in
 							// gno, never with reflect.
 						} else {
-							if isShift {
```

Comment:
that pr is another large one(?), and all it does is add the condition back in.
PRs should be complete, need to add back in in case the other PR isn't merged.

Reply:
not sure why a followup PR is needed; does it fix the failing tests here? (TestDebug)
isShift will never be true with this PR but #1775 adds this condition back? confusing.